### DISCUSSION TITLE
100+ question layout

### DISCUSSION THREAD

**Discussion by karsood** [Sunbird-Saral/Community#65] [Ideas]
Dear team, I am sharing below a requirement for Saral. Please do help with the creation of a layout.

Concept: State-wide standardized assessments for grades 1-9. Each class has different number of OMR based questions that need to be answered. This assessment is conducted annually and is of prime importance to the State. I am attaching the previously used layouts below for your reference.

Request: A Saral scannable design layout to be used by the state for the next iteration of the exam.

[Class - 2 OMR.pdf](https://github.com/Sunbird-Saral/Community/files/10433085/Class.-.2.OMR.pdf)
[Class_-_1_OMR.pdf](https://github.com/Sunbird-Saral/Community/files/10433086/Class_-_1_OMR.pdf)
[Class_-_3_5_OMR.pdf](https://github.com/Sunbird-Saral/Community/files/10433088/Class_-_3_5_OMR.pdf)
[Class_-_6_8_OMR.pdf](https://github.com/Sunbird-Saral/Community/files/10433089/Class_-_6_8_OMR.pdf)
[Class_-_9_OMR.pdf](https://github.com/Sunbird-Saral/Community/files/10433090/Class_-_9_OMR.pdf)
[School Evaluation Form.pdf](https://github.com/Sunbird-Saral/Community/files/10433091/School.Evaluation.Form.pdf)

**Comment by dileep-gadiraju** [comment]
@karsood , Created https://project-sunbird.atlassian.net/browse/SS-87. UX Designer started looking into it.

**Comment by Asiya-Bagwan** [comment]
Is the above template, which has a vertical size, supported in Saral?
